packages/webpack/src/presets/base.ts

```diff
@@ -230,6 +230,7 @@ function getEnv (ctx: WebpackConfigContext) {
     'process.env.NODE_ENV': JSON.stringify(ctx.config.mode),
     'process.mode': JSON.stringify(ctx.config.mode),
     'process.dev': options.dev,
+    __NUXT_VERSION__: JSON.stringify(ctx.nuxt._version),
```

Comment:
Maybe we could use `process.versions.nuxt`? (i know it is not best but since we are already practicing `process.` static shims, and also Node.js uses `process.versions` seems simpler also future proof to also shim runtime if needed)

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/5158436/222143333-53534fde-55ef-4b38-89ee-5c560d3f3f4b.png">

Reply:
What would be the benefit of this?

Reply:
`import.meta` seems a good idea to be replaced (however it also inherits the same concerns when we do!). Syntax of  `__NUXT_VERSION__` injection is almost the same as `globalThis. __NUXT_VERSION__` although it is internal even i was confused first time checking PR code.

Reply:
I understand it can be confusing. If you think of a better wording, I'm interested. But this is an internal setting - and, for example, here we already use non-existing env variables and a similar vue devtools flag:

https://github.com/nuxt/nuxt/blob/32f6cb27b7306ec3ff3f4ce7fbd1d60eaac47efd/packages/nuxt/src/core/nitro.ts#L141-L150

Listening to your comments, I think your concern isn't about technical implementation but just about the format of the string that gets replaced, which is an internal factor we aren't exposing publicly. I can see points in favour of both ` __NUXT_VERSION__` and `process.versions.nuxt` but would prefer not to piggyback on `process.versions` as we do nothing with `process.versions` so far and it further ties us to Node APIs.